### ISSUE TITLE
sset File Uploads Successfully But Not Shown In File List

### ISSUE THREAD

**Issue by Todd-H** [snipe/snipe-it#5126]
I just started working with Snipe-IT and my test imports showed the assets but after I deleted them and soft purged them now when I import new ones they do now show up in All Assets. I look in the status under setting and it shows 5 but when I click in there it is not showing the assets, it is saying no matching records found. I have looked under every option and they are not there. Even though there is only one company I have enabled multi-company like above.

•	Version of Snipe-IT you're running: v4.1.10 – build 3249
•	What OS and web server you're running Snipe-IT on: Windows Server 2012 R2
•	What method you used to install Snipe-IT (install.sh, manual installation, docker, etc): manual installation
•	If you're getting an error in your browser, include that error: N/A
•	What specific Snipe-IT page you're on, and what specific element you're interacting with to trigger the error:
•	If a stacktrace is provided in the error, include that too.: N/A
•	Any errors that appear in your browser's error console. N/A
•	Confirm whether the error is reproduceable on the demo:
•	Include any additional information you can find in app/storage/logs and your webserver's logs.: I presume this is now storage/logs.
•	Include what you've done so far in the installation, and if you got any error messages along the way.: N/A
•	Indicate whether or not you've manually edited any data directly in the database: No.

**Comment by snipe** [comment]
There isn’t such a thing as soft purging. Purging deletes permanently. Can you clarify what you did, and whether there are any records in your assets table, as well as your app log?

**Comment by Todd-H** [comment]
I removed all of the test assets from out of the Asset list then I click on purge under the main setting list to get rid of them out. Im not 100% sure if that is a soft purge or the other one. Which I think would be fine since they were test assets

Now I am adding in production assets and thats when they do not show in the list,  but in the Deployed status under the settings that I should have 5 assets. I click on the Deployed link it takes me to the asset list but it says "No Matching Records"

**Comment by Todd-H** [comment]
What is the limit of assets into the program? we have over 13,000 assets we need to track.

**Comment by snipe** [comment]
There is no limit. We have customers with over 100k assets.

**Comment by Todd-H** [comment]
Thank you for letting me know about the limit but do you have an answer for my issue. Here is the answers for you original question.

I removed all of the test assets from out of the Asset list then I click on purge under the main setting list to get rid of them out. Im not 100% sure if that is a soft purge or the other one. Which I think would be fine since they were test assets

Now I am adding in production assets and thats when they do not show in the list, but in the Deployed status under the settings that I should have 5 assets. I click on the Deployed link it takes me to the asset list but it says "No Matching Records"

**Comment by Todd-H** [comment]
I know you are extremely busy, but I never got an answer to my issue.

"I just started working with Snipe-IT and my test imports showed the assets but after I deleted them and soft purged them now when I import new ones they do now show up in All Assets. I look in the status under setting and it shows 5 but when I click in there it is not showing the assets, it is saying no matching records found. I have looked under every option and they are not there. Even though there is only one company I have enabled multi-company like above."
